scripts/swagger-gen.sh

```diff
@@ -29,6 +29,11 @@ swagger_operations+=("GetProjectsProjectID")
 swagger_operations+=("PutProjectsProjectID")
 swagger_operations+=("DeleteProjectsProjectID")
 
+swagger_operations+=("GetUsers")
+swagger_operations+=("PostUsers")
+swagger_operations+=("PutUsersUserterminaID")
```

Comment:
Please double check this, as `PutUsersUserterminaID` does not sound correct. Also, I do not see any usage of it.

Reply:
Yes that was a typo. Should have been `PutUsersUserID`. That typo sneaked in after a generated the code, so I only had to fix it in this script file.